packages/office-ui-fabric-react/src/components/Calendar/CalendarDay.tsx

```diff
@@ -123,7 +123,6 @@ export class CalendarDay extends BaseComponent<ICalendarDayProps, ICalendarDaySt
         <FocusZone>
           <table
```

Comment:
If you're using application, put role="presentation" on the table to signal that it's just for layout